workflow/parse.py

```diff
@@ -0,0 +1,64 @@
+import os 
+import pandas as pd 
+import csv
+ 
+tabb = dict()
+tabb['strand'] = []
+tabb['binding_position'] = []
+tabb['binding_posterior'] = []
+tabb['pwm_id'] = []
+tabb['binding_region'] = []
+tabb['binding_sequence'] = []
+tabb['binding_energy'] = []
+tabb['fast_record'] =[] 
```

Comment:
the format is called `FASTA`, it would be better to add the `a` and we have the full name here.